### ISSUE TITLE
Issue building image from dockerfile

### ISSUE THREAD

**Issue by JohnGiorgi** [BMDSoftware/neji#15]
I am having an issue building an image form the dockerfile. Namely, step 4/13 failed:

```
Step 4/13 : ADD neji-server.zip /opt/neji-server.zip
ADD failed: stat /var/lib/docker/tmp/docker-builder601862817/neji-server.zip: no such file or directory
```

I don't see `/opt/neji-server.zip` in the neji github directory, so I am not sure how to fix this issue. 

Admittedly I do not know much about Docker/Dockerfiles so perhaps I am making a silly mistake. Any advice for setting up this docker would be appreciated!

**Comment by davidcampos** [comment]
@JohnGiorgi 
The `neji-server.zip` file should be generated using the `neji.sh` utility with the `-s` argument flag. Than place the generated file in `/opt/neji-server.zip` or change the path in the Dockerfile to your file. After this you can build your own docker image.

**Comment by JohnGiorgi** [comment]
Thanks for the response David. 

I download the latest version of neji, move into the folder and call `./neji.sh -s`. I get the following error in return:

```
[ERROR] There was a problem processing the input arguments.
org.apache.commons.cli.UnrecognizedOptionException: Unrecognized option: -s
	at org.apache.commons.cli.Parser.processOption(Parser.java:363)
	at org.apache.commons.cli.Parser.parse(Parser.java:199)
	at org.apache.commons.cli.Parser.parse(Parser.java:85)
	at pt.ua.tm.neji.cli.Main.main(Main.java:157)
```

I get this error if I try the above steps but from within a cloned repo:

```
Exception in thread "main" java.lang.IllegalArgumentException: No enum constant pt.ua.tm.neji.context.InputFormat.raw
	at java.lang.Enum.valueOf(Enum.java:238)
	at pt.ua.tm.neji.context.InputFormat.valueOf(InputFormat.java:35)
	at pt.ua.tm.neji.cli.Main.main(Main.java:285)
```

Any advice?

**Comment by davidcampos** [comment]
@JohnGiorgi 
Please make sure you are using the latest version of neji: [2.0.1](https://github.com/BMDSoftware/neji/releases/tag/v2.0.1).
This version included some changes regarding the server generation.

**Comment by JohnGiorgi** [comment]
@davidcampos 

Sorry to be a pain, but I download the latest version (following your link above) and call the `neji.sh` script with the `-s` flag (and all the other arguments it requires) and I get the following Java Exception:

```
java.io.IOException: The neji-server jar file must be packaged first! Please run 'mvn clean install' first.
	at pt.ua.tm.neji.cli.Main.generateServer(Main.java:677)
	at pt.ua.tm.neji.cli.Main.main(Main.java:562)
```

This looks like a Maven command? I do not know much about Maven and so I do not know how to proceed with this error. 

Any advice? Thanks so much for your patience and help.

**Comment by davidcampos** [comment]
@JohnGiorgi 
Can you please post the exact command that you are running to generate the server package?

**Comment by JohnGiorgi** [comment]
@davidcampos 

Yup. So I call `./neji.sh -s -i example/annotate/in/ -if RAW -o example/annotate/out/ -of JSON` exactly and I get the error:
```
java.io.IOException: The neji-server jar file must be packaged first! Please run 'mvn clean install' first.
	at pt.ua.tm.neji.cli.Main.generateServer(Main.java:677)
	at pt.ua.tm.neji.cli.Main.main(Main.java:562)
```

If you are wondering why I am providing all the other arguments beside `-s` it is because Neji complains otherwise. 

Thanks again!

**Comment by davidcampos** [comment]
@JohnGiorgi 
I have released [Neji 2.0.2](https://github.com/BMDSoftware/neji/releases/tag/v2.0.2) fixing a minor issue with server generation.
Please test it and tell me if server generation is working properly.

**Comment by JohnGiorgi** [comment]
@davidcampos 
That seemed to do it! I was able to generate the .zip file.

Thanks so much for all of your help!
